userland/examples/radio_rx/main.c

```diff
@@ -0,0 +1,33 @@
+#include <stdbool.h>
+#include <stdio.h>
+#include <led.h>
+#include <radio.h>
+#include <timer.h>
+
+#define BUF_SIZE 60
+char packet_rx[BUF_SIZE];
+char packet_tx[BUF_SIZE];
+bool toggle = true;
+
+static void callback(__attribute__ ((unused)) int unused0,
```

Comment:
Fair enough. I don't have strong feelings about. I only brought it up in off-chance that people didn't know there was an alternative.